src/index.css

```diff
@@ -1,13 +1,339 @@
-body {
+body, html {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  overflow-y: hidden;
+  overflow-x:auto;
+  /*max-width: 100%;*/
+  padding: 0;
 }
 
+
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.input {
+  padding: 16px 0;
+  background: white;
+  width: 100%;
+}
+
+.inputButtons {
+  display: flex;
+  justify-content: space-between;
+}
+
+.attachIcon{
+  background-image: url("icons/attachIcon/attachIcon.svg");
+  background-size: cover;
+  padding: 1px;
+  width: 20px;
+  height: 20px;
+}
+
+.file-input-container {
+  position: relative;
+  display: inline-block;
+  cursor: pointer;
+}
+/* Hide the original file input */
+.file-input-container input[type="file"] {
+  position: absolute;
+  left: 0;
+  top: 0;
+  opacity: 0;
```

Comment:
Why `absolute`? Why `opacity: 0`? Please read about `visibility and opacity` VS `display: none` differences